ravenml/data/interfaces.py

```diff
@@ -23,6 +34,98 @@ class CreateInput(object):
     for getting additional information.
 
     """
+    def __init__(self, config:dict=None, plugin_name:str=None):
+
+        if config is None or plugin_name is None:
+            raise click.exceptions.UsageError(('You must provide the --config option '
+                'on `ravenml create` when using this plugin command.'))
+        
+        self.config = config
+        
+        ## Set up Local Cache
+        # TODO: maybe create the subdir here?
+        # currently the cache_name subdir is only created IF the plugin places files there
+        self.plugin_cache = RMLCache(plugin_name)
```

Comment:
I just realized this is going to cause a conflict with the cache being made by the train commands. Right now the plugin name for both is just `tf-bbox` because both the training plugin and the dataset plugin expose the command with that name, despite their python packages being named differently.

To fix this, we should automatically prepend `data_` to the plugin name here, and then prepend `train_` to the plugin name used in the train command. I will push a commit that does this.